include/aws/common/math.inl

```diff
@@ -150,6 +196,102 @@ AWS_STATIC_IMPL int aws_round_up_to_power_of_two(size_t n, size_t *result) {
 #    pragma warning(pop)
 #endif /* _MSC_VER */
 
+AWS_STATIC_IMPL uint8_t aws_min_u8(uint8_t a, uint8_t b) {
```

Comment:
These are all basically the same. Why not have a macro that generates them?

Reply:
I could see both ways. Going to keep it this way for the sake of simplicity/readability.